source/_docs/manual-import.md

```diff
@@ -123,7 +123,7 @@ sleep 180
 fi
 done
 ```
-This script connects to your Pantheon site's Dev environment and starts uploading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  
+This script connects to your Pantheon site's Dev environment and starts downloading your files. If an error occurs during transfer, rather than stopping, it waits 180 seconds and picks up where it left off.  

```

Comment:
@twooten downloading/uploading are both accurate, but in the context of imports my preference is uploading - or we could go with something like this: 
> This script connects to your Pantheon site's Dev environment and starts synchronizing your files.

Let me know what you think
cc @nataliejeremy